lms/static/sass/views/_login-register.scss

```diff
@@ -354,7 +358,11 @@
             }
 
             &.error {
-                border-color: $error-color;
+                border-color: $red;
```

Comment:
It's a part of the requirements (`$red` matches the requirements):

![](http://i.imgur.com/2nZiAy1.png)

![](http://i.imgur.com/0EtGdig.png)